package.yaml

```diff
@@ -141,6 +141,7 @@ executables:
     ghc-options:
       - -threaded
       - -flate-specialise
+      - -funbox-strict-fields
```

Comment:
While I was in there, I've been meaning to enable this flag. It more aggressively unboxes strict fields, like the `Config`.

Reply:
@expede what do you mean by:
> aggressively unboxes strict fields

Reply:
Haskell is lazy by default: nothing is evaluated until it's used. This is achieved by leaving a placeholder "box" pointer in place (i.e. Schrodinger's box: may be full or empty, but observing it forces it to become full).

You can tell GHC that some data should be strict, and on record fields that's the `!`. However, in many cases that placeholder pointer remains, which is a (small) layer of indirection. Turning this on says "anywhere that you _can_ toss that pointer, do so." The plain value in memory is now "unboxed".